tchannel/tornado/dispatch.py

```diff
@@ -160,27 +165,24 @@ def check_health(request, response, proxy):
             A function that gets called with ``Request``, ``Response``, and
             the ``proxy``.
 
-        :param broker:
-            Broker injects customized serializer and deserializer into
-            request/response object.
+        :param req_serializer:
+            Arg scheme serializer of this endpoint. It should be
+            ``RawSerializer``, ``JsonSerializer``, and ``ThriftSerializer``.
```

Comment:
Actually - I disagree with this. These need to be separate operations so they can fail separately.